portfolio/src/main/java/com/google/sps/servlets/RandomGameServlet.java

```diff
@@ -15,18 +15,34 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+@WebServlet("/random-game")
+public final class RandomGameServlet extends HttpServlet {
+
+    private List<String> games;
+
+  @Override
+  public void init() {
+    games = new ArrayList<>();
+    games.add("Legend of Zelda: Breath of the Wild");
+    games.add("Monster Hunter: World");
+    games.add("Animal Crossing: New Horizons");
+    games.add("Valorant");
+    games.add("Counter Strike: Global Offensive");
+    games.add("Super Smash Bros. Ultimate");    
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String gameName = games.get((int) (Math.random() * games.size()));
+
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println(gameName);
```

Comment:
If the content type is set to `"text/html"` above, probably should wrap this in a tag (`<p></p>` would be fine). Then you would need to modify your javascript to set `.innerHtml` instead of `.innerText`. Or, you can modify the content type here to `text/plain`.

Reply:
Changed content type to text/plain since /random-game is just an endpoint.